README.md

```diff
@@ -35,10 +35,40 @@ import slickeffect.implicits._
 import scala.concurrent.ExecutionContext.Implicits.global
 
 //the instances will be in implicit scope
-scala> Async[slick.dbio.DBIO]
-res0: Async[slick.dbio.package.DBIO] = slickeffect.DBIOAsync@434c179e
+scala> Sync[slick.dbio.DBIO]
+res0: Sync[slick.dbio.package.DBIO] = slickeffect.DBIOSync@434c179e
 ```
 
+### Cats Effect 1.x/2.x -> 3.x migration guide
+
+Cats Effect 3 changes the hierarchy of the type classes that are available, in a way that only lets us implement `Sync`.
+
+To work around that limitation and ensure you can still combine `Async` with `DBIO` _somehow_, a new method has been added: `liftToDBIO`.
+It returns an `F ~> DBIO` for any `UnsafeRun`nable `F[_]` (such as `IO`, given a runtime instance, which `IOApp` will happily deliver to you).
+This should be useful to anyone who was using `Async[DBIO]` previously - for example, if you were instantiating a Tagless Final algebra that required `Async`:
+
+```scala
+trait Client[F[_]] {
+  def call: F[Unit]
+}
+
+def asyncClient[F[_]: Async]: Client[F] = ...
+```
+
+It is now impossible to directly get a `Client[DBIO]`, because of the `Async` constraint. However, it is still possible to get there through e.g. `IO`:
+
+```scala
+val ioClient: Client[IO] = asyncClient[IO]
+
+implicit val clientFunctorK: FunctorK[Client] = Derive.functorK
+
+val dbioClient: Client[DBIO] = ioClient.mapK(slickeffect.liftToDBIO[IO])
+```
```

Comment:
The fact that this is valid makes me wonder if there's something missing from the new hierarchy.

Reply:
It's been the case before - e.g. you can instantiate something that requires `ConcurrentEffect` but `mapK` it into something that only goes up to `Async`, for example `StateT` (`liftF` from `IO`).